src/app/spotify-auth/spotify-auth.component.html

```diff
@@ -1,5 +1,16 @@
-<div *ngIf="userId; else logIntoSpotify">Spotify User ID: {{userId}}</div>
+<div *ngIf="userId; else logIntoSpotify" [className]="'user-id'"><b>Spotify User ID:</b> {{userId}}</div>
```

Comment:
I'm not familiar with this className syntax, why not `class="user-id"`?

Reply:
Done. My bad!